hash/gjrstar.xml

```diff
@@ -3,20 +3,60 @@
 <!--
 license:CC0-1.0
 -->
-<softwarelist name="gjrstar" description="VTech Genius Junior Redstar cartridges">
+<softwarelist name="gjrstar" description="VTech Genius Junior Redstar (and clones) cartridges">
 
 	<!-- According to the boxart, this is compatible with: Junior Redstar/Redstar 2/Redstar 3 -->
 	<software name="wortspie">
 		<description>Wortspiele</description>
 		<year>200?</year>
 		<publisher>Video Technology</publisher>
 		<info name="serial" value="80-10171" />
-
 		<part name="rom" interface="genius_rom">
 			<dataarea name="rom" size="0x40000">
 				<rom name="27-5883-00" size="0x40000" crc="99214bce" sha1="75294fbc411b0b39b3db4727824faf29127675af"/>
 			</dataarea>
 		</part>
 	</software>
 
+	<!-- Russian carts for Precomputer Electronik (Прекомпьютер Электроник), compatibility with clones from other regions is unknown -->
+
+	<software name="matematika">
+		<description>Matematika - Podarok Pervoklassniku</description> <!-- Mathematics - Gift for a First Grader -->
+		<year>????</year>
+		<publisher>Video Technology / DjiNo</publisher> <!-- VTech / ДжиНо -->
+		<info name="serial" value="80-10126-04" /> <!-- Nº 4 -->
```

Comment:
Please use the real numero sign character (U+2116 №)

Reply:
Fixed!